src/System.Net.Security/src/System/Net/SecureProtocols/SslState.cs

```diff
@@ -1576,13 +1577,14 @@ private Framing DetectFraming(byte[] bytes, int length)
 
             int version = -1;
 
-            if ((bytes == null || bytes.Length == 0))
+            if ((bytes == null || bytes.Length <= 0))
```

Comment:
I disagree with this as well: a Debug.Assert is meant to validate the contract internally. If we'd know that the contract is to have <0 values, the validation code shouldn't be `DEBUG`-only.

This code is to fix the incorrect Desktop port and I believe that we need to at least initially match the behaviors exactly. I actually don't agree with the change made to the original statement (to apply the negation) as the code diverged unnecessarily from desktop (the compiler should instead find the smallest number of Boolean operations anyway).